### ISSUE TITLE
Should the feed icon be on the left side of the Name/Author in the detail field?

### ISSUE THREAD

**Issue by peternlewis** [brentsimmons/NetNewsWire#590]
I'm finding I am not as aware of which feed an article comes from in NNW5 as I was in NNW3 - I don't really know why.

But I'm wondering if the feed icon should be on the left side (as in:

[Icon]  Daring Fireball
[Icon]  John Gruber

(double hight icon obviously, pardon my limited ASCIIArt).

As it stands, at least for me with a large monitor and a wide NNW three column window, the feed icon in the article is way off to the right out of primary vision.

Just a thought.

**Comment by brentsimmons** [comment]
I totally get it. And it’s what I did at first.

The problem in practice is that some feeds don’t have any artwork to show. So leaving a blank space, or using some kind of generic gray roundrect placeholder, just looked weird.

I also tried not leaving a blank space — just having the artwork-less articles flush left. And that was weird too.

It’s possible that the NetNewsWire artist (Brad Ellis) could make a generic icon of some kind for those that would look good, and that would solve the problem. But he’s working on the app icon.

In other words: this will get revisited at some point after 5.0 ships.

**Comment by peternlewis** [comment]
Cool, no worries.  Thanks for your consideration.
